Testscripts/Linux/TIMESYNC-NTP.sh

```diff
@@ -28,173 +28,161 @@ loopbackIP="127.0.0.1"              # IP to force NTPD to listen on IPv4
 # Source constants file and initialize most common variables
 UtilsInit
 
+GetDistro
 # Try to restart NTP. If it fails we try to install it.
-if is_fedora ; then
-    # RHEL 8 does not support NTP, skip test
-    if [[ $os_RELEASE =~ 8.* ]]; then
-        LogMsg "Info: $os_VENDOR $os_RELEASE does not support NTP. Test skipped. "
-        SetTestStateSkipped
-        exit 0
-    fi
-    # Check if ntpd is running
-    if ! service ntpd restart
-    then
-        LogMsg "Info: NTPD not installed. Trying to install..."
-        if ! yum install -y ntp ntpdate
-        then
-            LogErr "Unable to install ntpd. Aborting"
+case $DISTRO in
+        redhat_*|centos_*)
+            # RHEL 8 does not support NTP, skip test
+            if [[ $os_RELEASE =~ 8.* ]]; then
+                LogMsg "Info: $os_VENDOR $os_RELEASE does not support NTP. Test skipped. "
+                SetTestStateSkipped
+                exit 0
+            fi
+            # Check if ntpd is running
+            if ! service ntpd restart
+            then
+                LogMsg "Info: NTPD not installed. Trying to install..."
+                if [ ! yum install -y ntp ntpdate ]; then
```

Comment:
add [] will hit issue, that's why you see test fail on ubuntu

Reply:
Fixed